CHANGELOG.adoc

```diff
@@ -17,6 +17,7 @@ toc::[]
 === Added
 
 * CommonResourceLoad (util-test).
+* HttpMockObjectResponse (util-test).
```

Comment:
Does this mean that we need to import ktor everytime we import utils now? and that ktor versions should be compatible through our sdks?

Reply:
No - this is solved by #35...there I realised it is better to split those modules.